### DISCUSSION TITLE
How to increase the line spacing in the code?

### DISCUSSION THREAD

**Discussion by Archakov06** [code-hike/codehike#294] [General]
![image](https://user-images.githubusercontent.com/12086860/197461058-46aba3a8-a4c8-4a82-b153-9fa0a18b0690.png)

The code looks too compressed and I would like to make a small indent between the lines

I tried using line-height, but it turned out that the margins are set via the transform property

**Comment by pomber** [comment]
line-height used to work, I'll create an issue

**Reply by pomber** [reply]
#295
